pkg/system/reconciler.go

```diff
@@ -206,12 +210,16 @@ func NewReconciler(
 
 // CheckAll checks the state of all the objects controlled by the system
 func (r *Reconciler) CheckAll() {
+
 	CheckSystem(r.NooBaa)
 	util.KubeCheck(r.CoreApp)
-	util.KubeCheck(r.NooBaaDB)
 	util.KubeCheck(r.ServiceMgmt)
 	util.KubeCheck(r.ServiceS3)
-	util.KubeCheck(r.ServiceDb)
```

Comment:
why `util.KubeCheck(r.ServiceDb)` is not needed anymore?

Reply:
Fixed